tests/metadata.rs

```diff
@@ -566,6 +573,7 @@ const MANIFEST_OUTPUT: &'static str=
     }],
     "workspace_members": [ "foo 0.5.0 (path+file:[..]foo)" ],
     "resolve": null,
+    "target_directory": "[..]foo/target",
```

Comment:
The slashes won't work on windows. I think you can write `[..]foo[/]target` to select `\` or `/`.